src/stores/room-list/RoomListStore2.ts

```diff
@@ -63,7 +63,7 @@ export class RoomListStore2 extends AsyncStore<ActionPayload> {
 
         this.checkEnabled();
         for (const settingName of this.watchedSettings) SettingsStore.monitorSetting(settingName, null);
-        RoomViewStore.addListener(this.onRVSUpdate);
+        RoomViewStore.addListener(() => this.handleRVSUpdate({}));
```

Comment:
Could omit the object, but whatever you prefer.

Reply:
My IDE has other opinions on this matter, and I didn't really feel like fighting it today :p